### DISCUSSION TITLE
ser推理模型推理图片时，使用cpu推理，半精度推理，报错

### DISCUSSION THREAD

**Discussion by xiaoxianyu12** [PaddlePaddle/PaddleOCR#12748] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：paddlepaddle-gpu =2.6.1.post117; paddlenlp=               2.8.0
- 运行指令/Command Code：
- python kie/predict_kie_token_ser.py   \
--kie_algorithm=LayoutXLM   \
--ser_model_dir=../inference/ser_vi_layoutxlm/HRA2103COA_TLZ-100_UV928_6200EL_UV928RIANLON   \
--image_dir=../test_data/testSpeed/  \
--ser_dict_path=../train_data/HRA2103COA_TLZ-100_UV928_6200EL_UV928RIANLON.txt   \
--vis_font_path=../doc/fonts/simfang.ttf   \
--ocr_order_method="tb-yx"   \
--output=../outputInfer/ser/HRA2103COA_TLZ-100_UV928_6200EL_UV928RIANLON/testSpeed   \
--det_model_dir=../inference/ch_PP-OCRv4_det_server_infer   \
--rec_model_dir=../inference/ch_PP-OCRv4_rec_server_infer   \
--use_gpu =False \
--use_tensorrt=True \
--precision='fp16'
- 完整报错/Complete Error Message：
---    marked 696 operators to bfloat16
---    added 498 quantize ops before bfloat16 op
---    added 374 dequantize ops after bfloat16 op
---    squashed 0 scale with dequantize op
---    squashed 0 scale with quantize op
---    squashed 408 dequantize-quantize pairs
---    squashed 0 requantize ops
---    squashed 0 requantize ops
---    squashed 1 dequant with ops
---    squashed 0 quantize op
---    squashed 0 quantize with bfloat16 conv2d op
---    squashed 0 quantize with bfloat16 fused_conv2d op
Traceback (most recent call last):
  File "kie/predict_kie_token_ser.py", line 188, in <module>
    main(parse_args())
  File "kie/predict_kie_token_ser.py", line 156, in main
    ser_res, _, elapse = ser_predictor(img)
  File "kie/predict_kie_token_ser.py", line 120, in __call__
    self.predictor.run()
NotImplementedError: (Unimplemented) There are no kernels which are registered in the dequantize operator.
  [Hint: Expected kernels_iter != all_op_kernels.end(), but received kernels_iter == all_op_kernels.end().] (at ../paddle/fluid/framework/operator.cc:2273)
  [operator < dequantize > error]

**Comment by GreatV** [comment]
看着像 `dequantize` op还未实现或未正确注册相应的kernel。可以给 https://github.com/PaddlePaddle/Paddle/issues 提一个issue
